package.json

```diff
@@ -5,31 +5,41 @@
   "author": "Adam Klein <adam@500tech.com>",
   "homepage": "https://github.com/500tech/angular2-tree-component",
   "license": "MIT",
-  "main": "lib/angular2-tree",
-  "typings": "lib/defs/angular2-tree.d.ts",
+  "main": "angular2-tree-component",
+  "typings": "angular2-tree-component.d.ts",
   "keywords": ["angular2", "tree", "angular2-tree", "ng2", "ng2-tree"],
   "scripts": {
-    "test": "karma start"
+    "test": "karma start",
+    "clean": "rimraf node_modules && npm run clean:typescript && npm cache clean",
+    "clean:typescript": "rimraf angular2-tree.js angular2-tree.d.ts ./lib/**/*.js ./lib/components/*.d.ts ./lib/constants/*.d.ts ./lib/defs/api.d.ts ./lib/models/*.d.ts",
+    "build": "tsc",
+    "postinstall": "typings install"
   },
   "contributors": [
     "Adam Klein <adam@500tech.com>"
   ],
-  "peerDependencies": {
-    "@angular/common":  "2.0.0-rc.1",
-    "@angular/compiler":  "2.0.0-rc.1",
-    "@angular/core":  "2.0.0-rc.1",
-    "@angular/http":  "2.0.0-rc.1",
-    "@angular/platform-browser":  "2.0.0-rc.1",
-    "@angular/platform-browser-dynamic":  "2.0.0-rc.1",
-    "@angular/router":  "2.0.0-rc.1",
-    "@angular/router-deprecated":  "2.0.0-rc.1",
+  "dependencies": {
```

Comment:
I'm not familiar with peer dependencies. I got it working with dependencies
and didn't think any more of it. I'm fine with peer dependencies if they're
better.

On Wednesday, June 15, 2016, adam klein <notifications@github.com> wrote:

> In package.json
> <https://github.com/500tech/angular2-tree-component/pull/2#discussion_r67242672>
> :
>
> >    },
> >    "contributors": [
> >      "Adam Klein <adam@500tech.com <javascript:_e(%7B%7D,'cvml','adam@500tech.com');>>"
> >    ],
> > -  "peerDependencies": {
> > -    "@angular/common":  "2.0.0-rc.1",
> > -    "@angular/compiler":  "2.0.0-rc.1",
> > -    "@angular/core":  "2.0.0-rc.1",
> > -    "@angular/http":  "2.0.0-rc.1",
> > -    "@angular/platform-browser":  "2.0.0-rc.1",
> > -    "@angular/platform-browser-dynamic":  "2.0.0-rc.1",
> > -    "@angular/router":  "2.0.0-rc.1",
> > -    "@angular/router-deprecated":  "2.0.0-rc.1",
> > +  "dependencies": {
>
> As I understand, peerDependencies is better for angular2, because you
> don't want to bring multiple versions of Angular into the system, just use
> the one the user has provided in his/her package.json.
>
> Is there an advantage in using actual dependencies?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/500tech/angular2-tree-component/pull/2/files/0b1493bee41fcf3546b1d65ada5b9fcf52845f52#r67242672>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AA0lQPKwErxQSayM5lX2pDIgVV8-zrVeks5qMGUggaJpZM4Ilmsv>
> .
>